src/SDKs/Azure.Core/data-plane/Azure.Core/Azure.Core.csproj

```diff
@@ -0,0 +1,24 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+  <PropertyGroup>
+    <Description>This is the implementation of the Azure Client Pipeline</Description>
+    <AssemblyTitle>Microsoft Azure Client Pipeline</AssemblyTitle>
+    <Version>1.0.0-preview.1</Version>
+    <PackageTags>Microsoft Azure Client Pipeline</PackageTags>
+    <PackageReleaseNotes>
+      <![CDATA[
+      ]]>
+    </PackageReleaseNotes>
+
+    <!-- This is a workaorund until https://github.com/Azure/azure-sdk-for-net/issues/5214 is addressed -->
+    <RequiredTargetFrameworks>net461;netstandard2.0</RequiredTargetFrameworks>
+    
+    <TargetFrameworks>$(RequiredTargetFrameworks)</TargetFrameworks>
+    <LangVersion>7.2</LangVersion>
```

Comment:
Why do we need to explicitly set the langversion?

Reply:
We are using some C# language features not available in C# 7.0 (which is the default).